api/src/main/java/uk/gov/dvsa/mot/trade/api/response/searchvehicle/AnnualTestV1Response.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.dvsa.mot.trade.api.response.searchvehicle;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public final class AnnualTestV1Response extends AnnualTestResponse {
```

Comment:
I'd expect this to be v6 (the version specified in the request header), even though this is the first version of this particular response element. Maybe worth checking with the team, see what other people would expect.

This doesn't need to be resolved in the current pull request, for future reference only.

Reply:
**SearchVehicleV1Response**, **AnnualTestV1Response** and **DefectV1Response** build up **SearchVehicleV6ResponseMapper**.

Should in V7 (**SearchVehicleV7ResponseMapper**) we decide to update the ONLY VehicleResponse (**SearchVehicleV2Response**), we use the SearchVehicleV2Response, and V1 of the AnnualTest and Defect.

Mappers build up the response. And it is the mapper version which should be took as the API version.

**SearchVehicleV6ResponseMapper**
 - SearchVehicleV1Response
 - AnnualTestV1Response
 - DefectV1Response

**SearchVehicleV7ResponseMapper**
 - SearchVehicleV2Response
 - AnnualTestV1Response
 - DefectV1Response

This follows the existing pattern for *mot-tests* endpoints, and will enable a single version to be specified which affects both *mot-tests* and *annual-tests*.